pkg/api/backuptarget/healthy_handler.go

```diff
@@ -0,0 +1,80 @@
+package backuptarget
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+	"github.com/longhorn/backupstore"
+	"github.com/pkg/errors"
+	ctlcorev1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+
+	"github.com/harvester/harvester/pkg/config"
+	"github.com/harvester/harvester/pkg/controller/master/backup"
+	"github.com/harvester/harvester/pkg/generated/controllers/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/settings"
+	"github.com/harvester/harvester/pkg/util"
+)
+
+type HealthyHandler struct {
+	context      context.Context
+	settingCache v1beta1.SettingCache
+	secretCache  ctlcorev1.SecretCache
+}
+
+func NewHealthyHandler(scaled *config.Scaled) *HealthyHandler {
+	return &HealthyHandler{
+		context:      scaled.Ctx,
+		settingCache: scaled.HarvesterFactory.Harvesterhci().V1beta1().Setting().Cache(),
+		secretCache:  scaled.CoreFactory.Core().V1().Secret().Cache(),
+	}
+}
+
+func (h *HealthyHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
+	backupTargetName := mux.Vars(r)["backupTargetName"]
+	if backupTargetName != "default" {
+		util.ResponseError(rw, http.StatusBadRequest, fmt.Errorf("currently, only support default backup target"))
+		return
+	}
```

Comment:
do we need the backup target name since we only got one backup setting?

Reply:
Removed it. I think we can add this after we support multiple backup-target. Thank you.